content/en/docs/languages/net/exporters.md

```diff
@@ -5,60 +5,27 @@ weight: 50
 
 {{% docs/languages/exporters/intro dotnet %}}
 
-{{% alert title="Note" color="info" %}}
+### Dependencies {#otlp-dependencies}
 
-If you use the OpenTelemetry .NET Automatic Instrumentation for
-[automatic instrumentation](/docs/languages/net/automatic) you can learn how to
-setup exporters following the
-[Configuration Guide](/docs/languages/net/automatic/config)
+If you want to send telemetry data to an OTLP endpoint (like the
+[OpenTelemetry Collector](#collector-setup), [Jaeger](#jaeger) or
+[Prometheus](#prometheus)), you can choose between two different protocols to
+transport your data:
 
-{{% /alert %}}
+- HTTP/protobuf
+- gRPC
 
-## Console exporter
-
-The console exporter is useful for development and debugging tasks, and is the
-simplest to set up.
-
-```sh
-dotnet add package OpenTelemetry.Exporter.Console
-dotnet add package OpenTelemetry.Extensions.Hosting
-```
-
-If you're using ASP.NET Core, configure the exporter in your ASP.NET Core
-services:
-
-```csharp
-var builder = WebApplication.CreateBuilder(args);
-
-builder.Services.AddOpenTelemetry().WithTracing(b =>
-{
-    b.AddConsoleExporter()
-    // The rest of your setup code goes here too
-});
-```
-
-Otherwise, configure the exporter when creating a tracer provider:
-
-```csharp
-using var tracerProvider = Sdk.CreateTracerProviderBuilder()
-    .AddConsoleExporter()
-
-    // Other setup code, like setting a resource goes here too
-
-    .Build();
-```
-
-## OTLP endpoint
-
-To send trace data to an OTLP endpoint (like the [collector](/docs/collector) or
-Jaeger) you'll want to configure an OTLP exporter that sends to your endpoint.
+Start by installing the respective exporter package as a dependency for your
```

Comment:
nit: We can simply say "Install the OpenTelemetry.Exporter.OpenTelemetryProtocol package".. The "respective" package gives the impression that there are diff. exporter packages for HTTP/protobuf vs gRPC!